lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -732,6 +732,306 @@ class DecomposeAtenTriuOp : public OpRewritePattern<AtenTriuOp> {
 };
 } // namespace
 
+Value arange(PatternRewriter &rewriter, Location loc, Value size, Value dtype,
+             Value layout, Value device, Value pin_memory, Type type) {
+
+  auto arrangeType = getTensorTypeFromShapeValues({size}, type);
+  return rewriter.create<AtenArangeOp>(loc, arrangeType, size,
+                                       /*dtype=*/dtype, /*layout=*/layout,
+                                       /*device=*/device,
+                                       /*pin_memory=*/pin_memory);
+}
+
+void _getTrilSizes(PatternRewriter &rewriter, Location loc, int64_t row,
```

Comment:
nit: we don't follow the Python "underscore helper functions" convention here. `getTrilSizes` is fine. We do, however, set all helper functions as `static` functions